### PR TITLE
Define Entities as open classes

### PR DESCRIPTION
**Pull request by oskarikerppo**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -3,6 +3,7 @@ plugins {
     id("org.springframework.boot") version "2.5.6"
     id("org.jlleitschuh.gradle.ktlint") version "10.1.0"
     kotlin("jvm") version "1.5.31"
+    kotlin("plugin.allopen") version "1.4.32"
```
Comment (LeadingMoominExpert) [inline build.gradle.kts:6]: Kotlin version is wrong, should be 1.5.31

**Review by LeadingMoominExpert — CHANGES_REQUESTED** [review]
PR title should be more descriptive of what the changes are. Like here "Define Entities as open classes" would tell what's been done

**Comment by LeadingMoominExpert** [comment]
Also, don't make merge commits. Use rebase when updating main branch to your feature branch before the PR can be merged. 
https://www.atlassian.com/git/tutorials/merging-vs-rebasing
tl;dr
```bash
git checkout main && git pull && git checkout my-feature-branch && git rebase origin/main && git push --force-with-lease origin my-feature-branch
```
Interactive rebase is a good way to squash any WIP commits or some clutter if you want
```bash
git rebase -i origin/main
```

**Comment by oskarikerppo** [comment]
> PR title should be more descriptive of what the changes are. Like here "Define Entities as open classes" would tell what's been done

Changed the PR title as suggested

File: build.gradle.kts (hunk 1 above)
Comment (oskarikerppo) [reply build.gradle.kts]: kotlin version changed in commit e276a4b

**Review by LeadingMoominExpert — APPROVED** [review]
